src/ansys/templates/python/common/{{cookiecutter.__project_name_slug}}/.github/workflows/ci_cd.yml

```diff
@@ -45,10 +47,31 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: PyAnsys documentation style checks
-        uses: ansys/actions/doc-style@v7
+        uses: ansys/actions/doc-style@v10
         with:
           token: {{ '${{ secrets.GITHUB_TOKEN }}' }}
 
+  vulnerabilities:
+    name: Vulnerabilities
+    runs-on: ubuntu-latest
+    steps:
+      - name: PyAnsys Vulnerability check (on main)
+        if: github.ref == 'refs/heads/main'
+        uses: ansys/actions/check-vulnerabilities@v10
+        with:
+          python-version: {{ '${{ env.MAIN_PYTHON_VERSION }}' }}
+          python-package-name: {{ '${{ env.PACKAGE_NAME }}' }}
+          token: {{ '${{ secrets.PYANSYS_CI_BOT_TOKEN }}' }}
+
+      - name: PyAnsys Vulnerability check (on dev mode)
+        if: github.ref != 'refs/heads/main'
+        uses: ansys/actions/check-vulnerabilities@v10
+        with:
+          python-version: {{ '${{ env.MAIN_PYTHON_VERSION }}' }}
+          python-package-name: {{ '${{ env.PACKAGE_NAME }}' }}
+          token: {{ '${{ secrets.PYANSYS_CI_BOT_TOKEN }}' }}
+          dev-mode: true
```

Comment:
```suggestion
      - name: PyAnsys Vulnerability check
        if: github.ref == 'refs/heads/main'
        uses: ansys/actions/check-vulnerabilities@v10
        with:
          python-version: {{ '${{ env.MAIN_PYTHON_VERSION }}' }}
          python-package-name: {{ '${{ env.PACKAGE_NAME }}' }}
          token: {{ '${{ secrets.PYANSYS_CI_BOT_TOKEN }}' }}
          dev-mode: ${{ github.ref != 'refs/heads/main' }}
```

Reply:
Nice, @clatapie.

Reply:
Credits for this implementation go to @germa89 with [pymapdl](https://github.com/ansys/pymapdl/pull/3505/files/3cb1b93fd645401b940a881d37440458f42b2926#r1812742343)

Good catch @Revathyvenugopal162! I should have removed it, that was a typo from me